src/main/java/com/cloudbees/jenkins/support/filter/SensitiveContentFilter.java

```diff
@@ -52,9 +56,17 @@ public static SensitiveContentFilter get() {
     public @Nonnull String filter(@Nonnull String input) {
         ContentMappings mappings = ContentMappings.get();
         String filtered = input;
+        List<String> searchList = new ArrayList<>();
+        List<String> replacementList = new ArrayList<>();
+
         for (ContentMapping mapping : mappings) {
-            filtered = mapping.filter(filtered);
```

Comment:
Is the filter() method in mappings used anymore?

Reply:
I don't think so, but there are some lambda expressions where I cannot be 100% sure if it's used or not. That's why I'm not removing it.